cc/crypto/client_encryptor.cc

```diff
@@ -69,8 +70,10 @@ absl::StatusOr<EncryptedRequest> ClientEncryptor::Encrypt(absl::string_view plai
 
 absl::StatusOr<DecryptionResult> ClientEncryptor::Decrypt(EncryptedResponse encrypted_response) {
   // Decrypt response.
+  const std::vector<uint8_t> nonce(encrypted_response.encrypted_message().nonce().begin(),
```

Comment:
Is this copy required? Can't you just pass in encrypted_response.encrypted_message().nonce()?

Reply:
This copy will happen anyway, because the BoringSSL decryption function requires a vector

Reply:
It seem like it takes a pointer and a length: https://commondatastorage.googleapis.com/chromium-boringssl-docs/aead.h.html#EVP_AEAD_CTX_open. Our helper function takes a vector, but it seems like it doesn't have to: https://github.com/project-oak/oak/blob/78cba93869de13e6959c4b138f4e0fe9b1a6b3c3/cc/crypto/hpke/utils.cc#L130

Reply:
Will be done as part of https://github.com/project-oak/oak/issues/4600